tests/features/front-end/list-view/view-layout.feature

```diff
@@ -2,33 +2,61 @@ Feature: layout and visual elements of an event in List View
 
 Background:
     Given I am a Visitor on a site with TEC
-    Given List View is enabled under Events --> Settings --> Display
-    Given the site timezone is America/Los_Angeles
-    Given "Number of events to show per page" is set to 10
+    And List View is enabled under Events --> Settings --> Display
```

Comment:
Quotes around "List View"; here and everywhere else you might think of "variables" use quotes.
e.g. 
`And "List" view is enabled`
`And "Month" view is enabled`
`And "Another" view is enabled`

The tests are not going to consume the information "under Events --> Settings --> Display" as the tests will just set an option in the database; if we really want to embed the setting current location in the tests then I'd use a shorter notation like `Events > Settings > Display` or `Some > Other > Menu in the plugin`.

Reply:
@lucatume I've been using List View, etc. without quotation marks all over the place in other scenarios. Is it worth changing those, or should I just keep this in mind going forward? I'll try to remember to do setting paths like you suggest above in future scenarios.